onnxruntime/core/providers/webgpu/reduction/reduction_ops.cc

```diff
@@ -328,5 +339,24 @@ ReduceOpSpecificCode ReduceLogSumExp::GetOpSpecificCode(const Tensor* input_tens
   return code;
 }
 
+ReduceOpSpecificCode ArgMin::GetOpSpecificCode(const Tensor* input_tensor) const {
+  ORT_UNUSED_PARAMETER(input_tensor);
+  std::string loop_header = "var best_element = first_element; var best_index = u32(0);";
+  std::string loop_body = "if (current_element < best_element) { best_element = current_element; best_index = last_index; };";
+  std::string loop_footer = "let output_value = output_value_t(best_index);";
+  ReduceOpSpecificCode code({loop_header, loop_body, loop_footer});
+  return code;
+}
+
+ReduceOpSpecificCode ArgMax::GetOpSpecificCode(const Tensor* input_tensor) const {
+  ORT_UNUSED_PARAMETER(input_tensor);
+  std::string loop_header = "var best_element = first_element; var best_index = u32(0);";
```

Comment:
`best_index` is not 0s. It should `input_indices`. If max/min element is the first element along the axis in a multidimensional tensor then the all indices other than the index at axis need to populated from the for loop indices.